node/runtime/src/lib.rs

```diff
@@ -85,7 +85,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
 	spec_version: 162,
-	impl_version: 162,
+	impl_version: 163,
```

Comment:
Not required.

Reply:
I agree but the linter got mad b/c `core/sr-api-macros` changed.

In commit message: "Nothing at all actually changed, but core/sr-api-macros was updated to
pass a new arg to NativeExecutor and the linter things that requires an impl update."

Reply:
Yeah, just ignore it.